README.rst

```diff
@@ -36,15 +38,18 @@ Use pip to install pyfar
 (Requires Python 3.8 or higher)
 
 Audio file reading/writing is supported through `SoundFile`_, which is based on `libsndfile`_. On Windows and OS X, it will be installed automatically. On Linux, you need to install libsndfile using your distribution’s package manager, for example ``sudo apt-get install libsndfile1``.
+If the installation fails, please check out the `help section`_.
 
 Contributing
 ============
 
-Refer to the `contribution guidelines`_ for more information.
+Check out the `contributing guidelines`_ if you want to become part of pyfar.
```

Comment:
```suggestion
Check out the `contributing guidelines`_ if you want to become part of pyfar. For specific guidelines for this packge have a look in the pyfar documentation. 
```
maybe add a link to the local guidelines as well.

Reply:
This was actually a link to the pyfar specific guidelines. I removed the second sentence to be more clear